.github/workflows/fuzz.yml

```diff
@@ -53,8 +53,12 @@ hashes_sha1,
           override: true
           profile: minimal
       - name: fuzz
-        run: if [[ "${{ matrix.fuzz_target }}" =~ ^bitcoin ]]; then export RUSTFLAGS='--cfg=hashes_fuzz --cfg=secp256k1_fuzz'; fi
-        run: cd fuzz && ./fuzz.sh "${{ matrix.fuzz_target }}"
+        run: |
+          if [[ "${{ matrix.fuzz_target }}" =~ ^bitcoin ]]; then
+              export RUSTFLAGS='--cfg=hashes_fuzz --cfg=secp256k1_fuzz'
+          fi
+          echo "Using RUSTFLAGS "
```

Comment:
Did you want to print the RUSTFLAGS here?

Reply:
Lol, oops, I fixed this but forgot to push last night.